src/mpid/ch4/shm/xpmem/Makefile.mk

```diff
@@ -0,0 +1,27 @@
+## -*- Mode: Makefile; -*-
+## vim: set ft=automake :
+##
+## (C) 2016 by Argonne National Laboratory.
+## (C) 2014 by Mellanox Technologies, Inc.
+##     See COPYRIGHT in top-level directory.
```

Comment:
Just curious, but did Mellanox contribute code here or is this a copy paste? I've not been good about making sure the copyrights are all correct so maybe there's something more I should be doing.

Reply:
Good catch :)